pkgs/development/ocaml-modules/syslog/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, ocaml, findlib }:
+
```

Comment:
Please add an assertion making explicit that OCaml ≥ 4.03 is needed.

Reply:
True, because of `lowercase_ascii`. I did it.
Thank you!